### PR TITLE
Written doc install-porter-en.

### PR DESCRIPTION
**Pull request by Patrick-LuoYu**
Signed-off-by: Patrick-LuoYu <patrickluo@yunify.com>
/assign @FeynmanZhou @zheng1 @duanjiong

### CODE REVIEW

**Review by FeynmanZhou — CHANGES_REQUESTED** [review]

File: doc/install-porter.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Install Porter Using kubectl
+
+Log in to the Kubernetes cluster over SSH and run the following command:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/kubesphere/porter/master/deploy/porter.yaml
+```
+
+## Install Porter Using Helm Charts
```
[the remark this answers is not in the document]
Comment (duanjiong) [reply doc/install-porter.md:22]: I would recommend using helm3, but I have never used helm2 before. The current kubesphere installation is also helm3.

[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply doc/install-porter.md:22]: There should be two installation guides: 
- Installing Porter on Kubernetes (Kubectl and Helm)
- Installing Porter on KubeSphere (Web Console)

File: doc/install-porter-on-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Install Porter Using Helm
+
+Log in to the Kubernetes cluster over SSH and run the following commands:
+
+```bash 
+helm repo add test https://charts.kubesphere.io/test
+helm repo update
+helm install porter test/porter
+```
```
Comment (FeynmanZhou) [inline doc/install-porter-on-kubernetes.md:32]: @duanjiong Is it better to add the verification at the end of this guide? Users may not know whether the Porter is running normally or not.

File: doc/install-porter-on-kubesphere.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+6. Select **KubeSphere-test** from the drop-down list and click **porter**.
+
+   ![install-porter-1](./img/install-porter-6.jpg)
+
+7. Click **Deploy** and follow the wizard instructions to complete the installation. You can customize the chart configuration in the YAML file based on your requirements.
+
+   ![install-porter-1](./img/install-porter-7.jpg)
+
+   ![install-porter-1](./img/install-porter-8.jpg)
```
Comment (FeynmanZhou) [inline doc/install-porter-on-kubesphere.md:44]: @duanjiong Same here.

**Comment by FeynmanZhou** [comment]
I just noticed that all images are named with the serial number, I assume the images should be named with their related meaning. Do we need to add a rule for image naming in [Style Guide](https://github.com/kubesphere/website/blob/master/KubeSphere%20Documentation%20Style%20Guide.md#images)? @Sherlock113

**Comment by Patrick-LuoYu** [comment]
I think naming images with numbers is better, because those images are usually only related to the document. I cannot come up with a scenario where image names with actual meanings could be useful. On the other hand, names with actual meanings make document writing and localization difficult because the images are difficult to locate when there are a huge number of images. Naming images with chapter numbers (which we don't have) and sequence numbers is a common practice in document writing.

**Comment by zheng1** [comment]
Please fix the conflict file

**Comment by duanjiong** [comment]
You can overwrite mine directly in English, and I'll write Chinese at the back

**Pull request closed without merging**